tests/transform_dialect/cpu/BUILD.bazel

```diff
@@ -14,20 +14,12 @@ package(
 iree_lit_test_suite(
     name = "lit",
     srcs = [
-        "contraction-packing.mlir",
-        "contraction-packing-and-dispatch.mlir",
-        # DISABLED: incorrectly assuming default flag values.
-        # "eltwise_reduction_eltwise.mlir",
-        "fold_tensor_slice_into_transfer.mlir",
-        # DISABLED: incorrectly assuming default flag values.
-        # "matmul.mlir",
         "matmul_library_call.mlir",
```

Comment:
Should be able to drop `iree-benchmark-module` and `iree-opt` from the `tools = [ ...` here